### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by rrshidev**

### CODE REVIEW

File: 2-1-3.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+s = input()
+sum_k = len(s) * 60
+sum_r = float(sum_k / 100)
+sum_str = str(sum_r)
+sum_str_result = sum_str
+sum_str_result = sum_str_result.replace('.', ' р. ')
```
Comment (upvas) [inline 2-1-3.py:6]: No need to have this assignment, you can safely delete it:
```suggestion
sum_str_result = sum_str.replace('.', ' р. ')
```

Comment (upvas) [inline 2-1-3.py]: No need to have this assignment, you can safely delete it:

```suggestion
sum_str_result = sum_str.replace('.', ' р. ')
```

File: 2-1-3.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+if sum_str_result[-1] !=0:
```
Comment (upvas) [inline 2-1-3.py]: Missing whitespace

File: 2-1-3.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+s = input()
+sum_k = len(s) * 60
```
Comment (upvas) [inline 2-1-3.py]: Could you give these variables a more descriptive name, please?

File: 2-1-2.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+weight = float(input())
+growth = float(input())
```
Comment (upvas) [inline 2-1-2.py]: `height` would be a more appropriate name

File: 2-1-2.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+BMI = weight / (growth ** 2)
+
+if 18.5 <= BMI <= 25:
+    print('Оптимальная масса')
+elif BMI < 18.5:
+    print('Недостаточная масса')
+elif BMI > 25:
+    print('Избыточная масса')
```
Comment (upvas) [inline 2-1-2.py]: You can simply a little this if block:

```suggestion
if BMI < 18.5:
    print('Недостаточная масса')
elif BMI <= 25:
    print('Оптимальная масса')
else:
    print('Избыточная масса')
```

**Comment by upvas** [comment]
General comments:
- Please provide a short description of the pull request
- Use more descriptive variable names
- Add a Readme file to the repo
- Add a short description of the scripts in the beginning of the files reflecting what input it expects, what it does, and what it outputs

File: 2-1-4.py [hunk 6]
```diff
@@ -0,0 +1,3 @@
+#print(input().count(' ') + 1) - don't work =)
```
Comment (upvas) [inline 2-1-4.py]: What is this for? Usually it's better not to leave commented code lines - they just add noise.

File: 2-1-5.py [hunk 7]
```diff
@@ -0,0 +1,10 @@
+animal = ['Дракон', 'Змея', 'Лошадь', 'Овца', 'Обезьяна', 'Петух', 'Собака', 'Свинья', 'Крыса', 'Бык', 'Тигр', 'Заяц']
+per = 12
```
Comment (upvas) [inline 2-1-5.py]: What does this variable contain? Please give it a more descriptive name.

File: 2-1-5.py [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+year = int(input())
+index = (year + 4) % 12
+if index == 12:
+     index = 0
+
```
Comment (upvas) [inline 2-1-5.py]: Extra whitespaces.

**Review by upvas — CHANGES_REQUESTED** [review]

File: 2-1-6.py [hunk 9]
```diff
@@ -0,0 +1,18 @@
+numbers = int(input())
+s_num = str(numbers)
+if len(s_num) == 5:
+    s_num = s_num[-1::-1]
+    for i in s_num:
```
Comment (upvas) [inline 2-1-6.py]: you can also traverse a string backward like this:
```
for i in reversed(s_num)
```

File: 2-1-3.py (hunk 2 above)
Comment (rrshidev) [reply 2-1-3.py]: where?

File: 2-1-3.py (hunk 3 above)
Comment (rrshidev) [reply 2-1-3.py]: yes, master!

File: 2-1-3.py (hunk 2 above)
Comment (rrshidev) [reply 2-1-3.py]: ou, I see. Sorry, I will fix this

File: 2-1-2.py (hunk 4 above)
Comment (rrshidev) [reply 2-1-2.py]: yes, you're right

File: 2-1-4.py (hunk 6 above)
Comment (rrshidev) [reply 2-1-4.py]: I kick this comment! =)

File: 2-1-5.py (hunk 8 above)
Comment (rrshidev) [reply 2-1-5.py]: where? 6th string?

File: 2-1-6.py (hunk 9 above)
Comment (rrshidev) [reply 2-1-6.py]: Thank you teacher! This is a cool skill!

File: 2-1-3.py (hunk 1 above)
Comment (rrshidev) [reply 2-1-3.py]: fixed! ihha

File: 2-1-2.py [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+
+Формат выходных данных
+Программа должна вывести текст в соответствии с условием задачи.'''
+
+weight = float(input())
+height = float(input())
+
+BMI = weight / (height ** 2)
+
+if BMI <18.5:
```
Comment (upvas) [inline 2-1-2.py]: Missing whitespace!

File: 2-1-3.py [hunk 11]
```diff
@@ -0,0 +1,19 @@
+'''Дана строка текста. Напишите программу для подсчета стоимости строки, исходя из того, что один любой символ (в том числе пробел) стоит 606060 копеек. Ответ дайте в рублях и копейках в соответствии с примерами.
+
+Формат входных данных
+На вход программе подается строка текста.
+
+Формат выходных данных
+Программа должна вывести стоимость строки.'''
+
+inputString = input()
+priceStringOfKopecks = len(inputString) * 60
```
Comment (upvas) [inline 2-1-3.py]: Long names are not good either! + Use snake_case in Python.
I would name it `price_kopecks`, `price_rubs`,  `result_price_str`

File: 2-1-6.py [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+Дано пятизначное или шестизначное натуральное число. Напишите программу, которая изменит порядок его последних пяти цифр на обратный.
+Формат входных данных
+На вход программе подается одно натуральное пятизначное или шестизначное число.
+
+Формат выходных данных
+Программа должна вывести число, которое получится в результате разворота, указанного в условии задачи. Число нужно выводить без незначащих нулей.
+    '''
+numbers = int(input())
+s_num = str(numbers)
+if len(s_num) == 5:
```
Comment (upvas) [inline 2-1-6.py]: Can you handle both cases (for len=5 and len=6) together?

File: 2-1-3.py [hunk 13]
```diff
@@ -0,0 +1,19 @@
+'''Дана строка текста. Напишите программу для подсчета стоимости строки, исходя из того, что один любой символ (в том числе пробел) стоит 606060 копеек. Ответ дайте в рублях и копейках в соответствии с примерами.
```
Comment (upvas) [inline 2-1-3.py]: "стоит 606060 копеек" - is this correct or it should be 60?

File: 2-1-3.py [hunk 14]
```diff
@@ -0,0 +1,19 @@
...
+
+Формат входных данных
+На вход программе подается строка текста.
+
+Формат выходных данных
+Программа должна вывести стоимость строки.'''
+
+inputString = input()
+priceStringOfKopecks = len(inputString) * 60
+priceStringOfRubles = float(priceStringOfKopecks / 100)
```
Comment (upvas) [inline 2-1-3.py]: For rubles, you can use integer division `//`, and for kopecks use the remainder of the decision `%`.
Then you can format it via format: `"{} р. {} коп.".format(rubles, kopecks)`

File: 2-2-1.py [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+Формат входных данных
+В первой строке записано количество точек. Каждая следующая строка состоит из двух целых чисел — координат точки (сначала абсцисса – xxx, затем ордината – yyy), разделенных символом пробела.
+
+Формат выходных данных
+Программа должна вывести количество точек, лежащих в каждой координатной четверти, как в примерах.
+
+Примечание. Учтите, что точки, лежащие на осях координат, не принято относить к какой-либо координатной четверти.
+'''
+
+numPoint = int(input())
```
Comment (upvas) [inline 2-2-1.py]: should be snake_case

File: 2-1-3.py (hunk 14 above)
Comment (upvas) [reply 2-1-3.py]: \+ should be snake_case

File: 2-2-12.py [hunk 16]
```diff
@@ -0,0 +1,24 @@
...
+Напишите функцию func(num1, num2), принимающую в качестве аргументов два натуральных числа num1 и num2 и возвращающую значение
+True если число num1 делится без остатка на число num2 и False в противном случае.
+Результатом вывода программы должно быть "делится" (если функция func() вернула True) и "не делится" (если функция func()
+вернула False).
+'''
+
+
+#the names of these variables and functions are dictated by the conditions of the task
+def func(num1, num2):
+    if num1 % num2 == 0:
```
Comment (upvas) [inline 2-2-12.py]: You can simplify this function to just one line: `return num1 % num2 == 0`

File: 2-2-12.py [hunk 17]
```diff
@@ -0,0 +1,24 @@
+'''
+Предикат делимости
+Напишите функцию func(num1, num2), принимающую в качестве аргументов два натуральных числа num1 и num2 и возвращающую значение
+True если число num1 делится без остатка на число num2 и False в противном случае.
+Результатом вывода программы должно быть "делится" (если функция func() вернула True) и "не делится" (если функция func()
+вернула False).
+'''
+
+
+#the names of these variables and functions are dictated by the conditions of the task
```
Comment (upvas) [inline 2-2-12.py]: Whitespace after `#`

File: 2-2-12.py [hunk 18]
```diff
@@ -0,0 +1,24 @@
...
+def func(num1, num2):
+    if num1 % num2 == 0:
+        return True
+    else:
+        return False
+
+
+num1, num2 = int(input()), int(input())
+
```
Comment (upvas) [inline 2-2-12.py]: Unnecessary whitespaces.

File: 2-2-3.py [hunk 19]
```diff
@@ -0,0 +1,31 @@
...
+Формат входных данных
+На вход программе подается строка текста, содержащая натуральные числа, разделенные пробелами.
+
+Формат выходных данных
+Программа должна вывести измененный список, разделяя его элементы одним пробелом.
+
+'''
+
+
+inputString = input().split(' ')
```
Comment (upvas) [inline 2-2-3.py]: 1. snake_case
2. give the variables more meaningful names.

File: 2-2-3.py [hunk 20]
```diff
@@ -0,0 +1,31 @@
...
+nums, outputList, outputList_, resultList = [], [], [], []
+
+for i in inputString:
```
Comment (upvas) [inline 2-2-3.py]: 1. `i` again!
2. no need to convert a string to a list of chars in your case - you can access string chars by an index in the same way as in a list.

File: 2-2-3.py [hunk 21]
```diff
@@ -0,0 +1,31 @@
...
+    nums.append(i)
+for i in range(1, len(nums), 2):
```
Comment (upvas) [inline 2-2-3.py]: I believe you can do it in place - convert a string to a list and replace adjacent elements right there.

File: 2-2-5.py [hunk 22]
```diff
@@ -0,0 +1,20 @@
...
+На вход программе подается строка текста, содержащая натуральные числа, разделенные символом пробела, расположенные по неубыванию.
+
+Формат выходных данных
+Программа должна вывести одно число – количество различных элементов списка.
+
+Примечание. Задачу можно решить без множеств.
+'''
+
+
+stringInput = input().split(' ')
```
Comment (upvas) [inline 2-2-5.py]: snake_case

File: 2-2-7.py [hunk 23]
```diff
@@ -0,0 +1,22 @@
...
+На вход программе подаются две строки текста, содержащие слова "камень", "ножницы" или "бумага".
+На первой строке записан выбор Тимура, на второй – выбор Руслана.
+
+Формат выходных данных
+Программа должна вывести результат жеребьёвки, то есть кто победит: Тимур, Руслан или же они сыграют вничью.
+
+Примечание. Правила игры стандартные: камень побеждает ножницы, но проигрывает бумаге, а ножницы побеждают бумагу.
+'''
+
+Tim, Rus = input(), input()
```
Comment (upvas) [inline 2-2-7.py]: better names would be `tim_choice`, and `rus_choice` - avoid using Capitalized variable names.

**Review by upvas — COMMENTED** [review]
1. Fix please all camelCase names to snake_case.
2. Delete .idea folder

File: 2-2-3.py (hunk 21 above)
Comment (rrshidev) [reply 2-2-3.py]: Сorrected all remarks up to this point in this script. And didn't see the obvious. But now I see it. And the truth is - I can easily do everything in one cycle. I will do so.

File: 2-2-7.py (hunk 23 above)
Comment (rrshidev) [reply 2-2-7.py]: it very strange. I think that we are corrected it together earler.

File: 2-2-6.py [hunk 24]
```diff
@@ -0,0 +1,35 @@
...
+Формат выходных данных
+Программа должна вывести «ДА» или «НЕТ» в соответствии с условием задачи.
+
+Примечание 1. Само на себя число из набора умножиться не может, другими словами, два множителя должны иметь разные индексы в наборе.
+
+Примечание 2. Для решения задачи используйте вложенные циклы.
+'''
+
+n = int(input())
+stringInputList = [input() for i in range(0, n)]
```
Comment (upvas) [inline 2-2-6.py]: 1. snake_case
2. `i` again
3. the name is cumbersome, I would suggest naming it just `nums`

File: 2-2-6.py [hunk 25]
```diff
@@ -0,0 +1,35 @@
...
+mul = int(input())
+
+
+def multi_check():
+    result = 'НЕТ'
+    for i in range(n):
+        x = int(stringInputList[i])
+        for j in range(n):
```
Comment (upvas) [inline 2-2-6.py]: you can start this loop from i+1

File: 2-2-8.py [hunk 26]
```diff
@@ -0,0 +1,30 @@
...
+Формат входных данных
+На вход программе подается строка текста, содержащая натуральные числа, разделенные символом пробела, расположенные по неубыванию.
+
+Формат выходных данных
+Программа должна вывести одно число – количество различных элементов списка.
+
+Примечание. Задачу можно решить без множеств.
+'''
+
+Tim, Rus = input(), input()
```
Comment (upvas) [inline 2-2-8.py]: Same thing as in the prev task.

File: 2-1-7.py [hunk 27]
```diff
@@ -0,0 +1,11 @@
+'''Standard American Convention
+
+На вход программе подаётся натуральное число. Напишите программу, которая вставляет в заданное число запятые в соответствии со стандартным американским соглашением о запятых в больших числах.
+
+Формат входных данных
+На вход программе подаётся натуральное число n, (0<n<10100)n, \, (0 < n < 10^{100})n,(0<n<10100).
+
+Формат выходных данных
+Программа должна вывести число с запятыми в соответствии с условием задачи.'''
+x = int(input())
```
Comment (upvas) [inline 2-1-7.py]: missing whitespace

File: 2-2-11.py [hunk 28]
```diff
@@ -0,0 +1,59 @@
...
+        if letter in sing_txt.strip():
+            print(sing_txt + x)
+            sing_txt = sing_txt.replace(letter, '').replace('  ', ' ')
+
+
+# INPUT
+sing_str = input() + ' запретил букву'
+sing_sym = []
+
+for i in range(1072, 1104):
```
Comment (upvas) [inline 2-2-11.py]: you can also use `ord` to make it clear what's the range of the alphabet:
```
range(ord('а'), ord('я') + 1)
```

File: 2-2-11.py [hunk 29]
```diff
@@ -0,0 +1,59 @@
...
+росоор прт уу о
+рср прт уу п
+рср рт уу р
+с т уу с
+т уу т
+уу у
+'''
+
+
+# MAIN FUNCTION
```
Comment (upvas) [inline 2-2-11.py]: nit: caps looks weird

File: 2-1-3.py [hunk 30]
```diff
@@ -0,0 +1,19 @@
+'''Дана строка текста. Напишите программу для подсчета стоимости строки, исходя из того, что один любой символ (в том числе пробел) стоит 606060 копеек. Ответ дайте в рублях и копейках в соответствии с примерами.
+
+Формат входных данных
+На вход программе подается строка текста.
+
+Формат выходных данных
+Программа должна вывести стоимость строки.'''
+
+inputString = input()
```
Comment (upvas) [inline 2-1-3.py]: snake case

File: 2-2-6.py (hunk 24 above)
Comment (rrshidev) [reply 2-2-6.py]: Here I use 'i' because it index. Did you see RANGE?

Comment (rrshidev) [reply 2-2-6.py]: but I can

**Review by upvas — APPROVED** [review]
Well done! 🎉

**Pull request merged**